### PR TITLE
Cln API

### PR DESCRIPTION
**Pull request by roeierez**
This PR moves from the greenlight.proto deprecated API and uses the node.proto instead.
Greenlight are going to retire the greenlight.proto very soon: https://github.com/Blockstream/greenlight/pull/279

### CODE REVIEW

**Comment by ok300** [comment]
Fixes #456

**Comment by ok300** [comment]
Also FYI, the CLN API sometimes runs into this: https://github.com/Blockstream/greenlight/issues/280

File: libs/sdk-core/src/persist/channels.rs [hunk 1]
```diff
@@ -80,22 +84,21 @@ impl SqliteStorage {
                    spendable_msat, 
                    receivable_msat,
                    closed_at,
-                   funding_outnum
+                   funding_outnum,
+                   closed_at,
```
Comment (dangeross) [inline libs/sdk-core/src/persist/channels.rs:88]: Inserting `closed_at` twice

File: libs/sdk-core/src/persist/channels.rs [hunk 2]
```diff
@@ -108,6 +111,9 @@ impl SqliteStorage {
                     _ => c.closed_at,
                 },
                 c.funding_outnum,
+                c.closed_at,
```
Comment (dangeross) [inline libs/sdk-core/src/persist/channels.rs:114]: `closed_at` also set in the match statement above:
```
match c.state {
    ChannelState::Opened | ChannelState::PendingOpen => None,
    _ => c.closed_at,
},
```

**Review by dangeross — CHANGES_REQUESTED** [review]
Looks good. Just picked up on some issues inserting/updating channels

File: libs/sdk-core/src/persist/channels.rs (hunk 1 above)
Comment (roeierez) [reply libs/sdk-core/src/persist/channels.rs]: Good one! Done.

File: libs/sdk-core/src/persist/channels.rs (hunk 2 above)
Comment (roeierez) [reply libs/sdk-core/src/persist/channels.rs]: Fixed

File: libs/sdk-core/src/greenlight/node_api.rs [hunk 3]
```diff
@@ -566,26 +558,24 @@ impl NodeAPI for Greenlight {
         Ok(())
     }
 
-    async fn sweep(
-        &self,
-        to_address: String,
-        fee_rate_sats_per_vbyte: u64,
-    ) -> Result<WithdrawResponse> {
-        let mut client = self.get_client().await?;
+    async fn sweep(&self, to_address: String, fee_rate_sats_per_vbyte: u64) -> Result<Vec<u8>> {
```
Comment (JssDWt) [inline libs/sdk-core/src/greenlight/node_api.rs:561]: Let's make `fee_rate_sats_per_vbyte` a `u32`.

**Review by JssDWt — COMMENTED** [review]
Looks good.
Just a nit about the fee rate conversion.

File: libs/sdk-core/src/greenlight/node_api.rs (hunk 3 above)
Comment (roeierez) [reply libs/sdk-core/src/greenlight/node_api.rs]: Done. Had to do it all the way through to the udl file.

**Review by JssDWt — APPROVED** [review]

**Pull request merged**